packages/core/admin/admin/src/pages/ProfilePage/components/Preferences/index.js

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  Typography,
+  Box,
+  Grid,
+  GridItem,
+  Flex,
+  Select,
+  Option
+} from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+import upperFirst from 'lodash/upperFirst';
+
+const DocumentationLink = styled.a`
+  color: ${({ theme }) => theme.colors.primary600};
+`;
+
+const Preferences = ({ onChange, values, localeNames, allApplicationThemes }) => {
+  const { formatMessage } = useIntl();
+  const themesToDisplay = Object.keys(allApplicationThemes).filter(
+    (themeName) => allApplicationThemes[themeName]
+  );
+
+  return (
+    <Box
+      background="neutral0"
+      hasRadius
+      shadow="filterShadow"
+      paddingTop={6}
+      paddingBottom={6}
+      paddingLeft={7}
+      paddingRight={7}
+    >
+      <Flex direction="column" alignItems="stretch" gap={4}>
+        <Flex direction="column" alignItems="stretch" gap={1}>
+          <Typography variant="delta" as="h2">
+            {formatMessage({
+              id: 'Settings.profile.form.section.experience.title',
+              defaultMessage: 'Experience',
+            })}
+          </Typography>
+          <Typography>
+            {formatMessage(
+              {
+                id: 'Settings.profile.form.section.experience.interfaceLanguageHelp',
+                defaultMessage:
+                  'Preference changes will apply only to you. More information is available {here}.',
+              },
+              {
+                here: (
+                  <DocumentationLink
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://docs.strapi.io/developer-docs/latest/development/admin-customization.html#locales"
+                  >
+                    {formatMessage({
+                      id: 'Settings.profile.form.section.experience.here',
+                      defaultMessage: 'here',
+                    })}
+                  </DocumentationLink>
```

Comment:
Would it be possible to use 
```suggestion
                  <Box
                    as="a"
                    color="primary600"
                    target="_blank"
                    rel="noopener noreferrer"
                    href="https://docs.strapi.io/developer-docs/latest/development/admin-customization.html#locales"
                  >
                    {formatMessage({
                      id: 'Settings.profile.form.section.experience.here',
                      defaultMessage: 'here',
                    })}
                  </DocumentationLink>
```

instead? This way you can get rid of the styled overwrite.